cmd/thanos/main.go

```diff
@@ -122,6 +122,9 @@ func main() {
 			cancel()
 		})
 	}
+
+	level.Warn(logger).Log("msg", "log.request.decision flag would be soon deprecated in the next release. Use request.logging instead.")
```

Comment:
What's this? Why to log this on every component run?

Reply:
Okay, I can remove the warning if it's too verbose